sgv2-graphqlapi/src/main/java/io/stargate/sgv2/graphql/impl/GraphqlServiceServerConfiguration.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.graphql.impl;
+
+import io.dropwizard.Configuration;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GraphqlServiceServerConfiguration extends Configuration {
+  public StargateConfiguration stargate = new StargateConfiguration();
+
+  /** Value class definitions */
+  static class StargateConfiguration {
+    public BridgeConfiguration bridge = new BridgeConfiguration();
+  }
+
+  public static class BridgeConfiguration {
+
+    private static final Logger LOG = LoggerFactory.getLogger(BridgeConfiguration.class);
+
+    public String host = "localhost";
+    public int port = 8091;
+
+    // !!! TODO: probably should default to true
+    public boolean useTls = false;
```

Comment:
so how is this overridden?

Reply:
Dropwizard options can be overridden with system properties, like this:
```
-Ddw.stargate.bridge.host=127.0.0.2
```
I think we should revisit configuration to unify the different approaches. Between Dropwizard, command-line options in `*Starter` classes, and random system properties in the code, it is getting hard to follow. I'll create a separate ticket.

Reply:
This value is also potentially overridden in `config.yaml` (like `graphqlapi/src/main/resources/config.yaml`) for what that is worth, DropWizard has 3 level configuration settings.

And yes, it would be good to unify these; sounds like something that would go naturally with Better Framework upgrade.

Reply:
Yea I missed that `extends Configuration`.. Resolving this.